src/ray/raylet/local_object_manager.cc

```diff
@@ -85,36 +85,104 @@ void LocalObjectManager::FlushFreeObjectsIfNeeded(int64_t now_ms) {
   }
 }
 
+int64_t LocalObjectManager::SpillObjectsOfSize(int64_t num_bytes_required) {
+  if (RayConfig::instance().object_spilling_config().empty() ||
```

Comment:
Btw, isn't the plasma store and object store running in a separate thread? If so, don't we need to lock this method? (since it is called within store.cc)?

Reply:
Oh hmm I think you're right about that. I'll add a lock. It seems weird that this wasn't triggered in any of the tests yet.